analyzer/src/main/kotlin/curation/SimplePackageCurationProvider.kt

```diff
@@ -20,16 +20,15 @@
 package org.ossreviewtoolkit.analyzer.curation
 
 import org.ossreviewtoolkit.analyzer.PackageCurationProvider
-import org.ossreviewtoolkit.model.Identifier
 import org.ossreviewtoolkit.model.Package
 import org.ossreviewtoolkit.model.PackageCuration
 
 /**
  * A [PackageCurationProvider] that provides the specified [packageCurations].
  */
 open class SimplePackageCurationProvider(val packageCurations: List<PackageCuration>) : PackageCurationProvider {
-    override fun getCurationsFor(packages: Collection<Package>): Map<Identifier, List<PackageCuration>> =
-        packages.mapNotNull { pkg ->
-            packageCurations.filter { it.isApplicable(pkg.id) }.takeUnless { it.isEmpty() }?.let { pkg.id to it }
-        }.toMap()
+    override fun getCurationsFor(packages: Collection<Package>): List<PackageCuration> =
+        packageCurations.filter { curation ->
+            packages.any { pkg -> curation.isApplicable(pkg.id) }
+        }.distinct()
```

Comment:
I believe `distinct()` should not be added here to more closely resemble the previous code, and because it would be dropped in the next commit anyway.